docs/guides.md

```diff
@@ -58,10 +58,30 @@ The user therefore needs some coins ([TLC](https://explore.tlbc.trustlines.found
 
 ```javascript
 const laika = new TLNetwork({
-  protocol: 'https',
-  wsProtocol: 'wss',
-  host: 'relay0.testnet.trustlines.network',
-  path: '/api/v1',
+  relayProviderApiUrl: {
+    protocol: 'https',
+    wsProtocol: '',
```

Comment:
Why do we now have wsProtocol in here and also relayProviderWsUrl? that does not make sense. Also the names are super long. An external does not now about provider or so. He probably does not even care that we use websockets or an api at all.
Please just use: `relayApiUrl` and `messagingApiUrl` or even just `relayUrl`.
and if someone provides the url with `https` then wie use `wss` for the websocket, and if he uses `http` we use `ws`